packages/gatsby-plugin-guess-js/README.md

```diff
@@ -36,6 +36,8 @@ module.exports = {
       options: {
         // Find the view id in the GA admin in a section labeled "views"
         GAViewID: `VIEW_ID`,
+        // Add JWT token to perform authentication on deployment builds
+        jwt: GA_JWT
```

Comment:
Could you explain what GA_JWT holds?

For gatsbyjs.org we're doing the following to make it work:
```
jwt: {
  client_email: process.env.ANALYTICS_SERVICE_ACCOUNT,
  private_key: process.env.ANALYTICS_SERVICE_ACCOUNT_KEY,
},
```

ENV vars look like:
```
ANALYTICS_SERVICE_ACCOUNT="<name>@<project>.iam.gserviceaccount.com"
```
```
ANALYTICS_SERVICE_ACCOUNT_KEY="-----BEGIN PRIVATE KEY-----\nPRIVATE KEY STRING\n-----END PRIVATE KEY-----"
```

Reply:
very good point, tried to follow the rabbit hole into Guess webpack and ga packages but haven't been able to work out where this goes yet. I can see that client_email, private_key combo in `www/` but it appears to be commented out currently?